packages/stellar-axelar-std-derive/src/upgradable.rs

```diff
@@ -42,47 +21,44 @@ pub fn upgradable(name: &Ident, args: MigrationArgs) -> TokenStream2 {
             }
         }
 
+        type __MigrationData = <#name as stellar_axelar_std::interfaces::CustomMigratableInterface>::MigrationData;
```

Comment:
```suggestion
        type __#name_MigrationData = <#name as stellar_axelar_std::interfaces::CustomMigratableInterface>::MigrationData;
```
Or add contract name for uniqueness